deploy/charts/external-secrets/README.md

```
@@ -218,8 +218,10 @@ The command removes all the Kubernetes components associated with the chart and
 | webhook.certManager.cert.create | bool | `true` | Create a certificate resource within this chart. See https://cert-manager.io/docs/usage/certificate/ |
 | webhook.certManager.cert.duration | string | `"8760h0m0s"` | Set the requested duration (i.e. lifetime) of the Certificate. See https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1.CertificateSpec One year by default. |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)